tests/py/unit/test_tools_build_utils.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from mat3ra.esse.models.core.reusable.axis_enum import AxisEnum
+
+from mat3ra.made.tools.build.utils import stack_two_materials
+from mat3ra.made.material import Material
+
+
+def test_stack_two_materials_z_direction():
+    """Test stacking two materials along z direction"""
+    material1 = Material.create_default()
+    material2 = Material.create_default()
```

Comment:
This should be done above or in fixtures. We should have some more realistic examples here - with a graphene and nickel or so

Reply:
We should address this